common/messaging/kafka/partitionAckManager_test.go

```diff
@@ -0,0 +1,110 @@
+// The MIT License (MIT)
```

Comment:
let's also rename the touched files to snake_case (or the whole folder).

Reply:
Sure, I will rename the other files when I write test for them.